### ISSUE TITLE
Add installation over NixOS

### ISSUE THREAD

**Issue by Chemaclass** [phel-lang/phel-lang.org#100]
Recently, we managed to install `phel:0.16` on NixOS
- https://github.com/NixOS/nixpkgs/pull/314348 by @drupol

The task is about adding the information how to use phel within nixos, either in an installation page or better in a new section within the getting started page: https://phel-lang.org/documentation/getting-started/

**Comment by drupol** [comment]
I'll review this at the end of the day, I'm not supposed to work on nix related stuff during working hours.

**Comment by Chemaclass** [comment]
Absolutely, no rush! Let me know if you need any help for this, @drupol 😄
